CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.10)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)   # For clang-tidy.
-set(BUILD_SHARED_LIBS ON)
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
libfmt's linker behavior is controlled by this option, so we have to set it as OFF if we want to link everything statically.

Reply:
As a general suggestion, IMO comments like this belong next to the code (where people will see them) and not on the github PR (where people won't)